src/styles/button.scss

```diff
@@ -0,0 +1,21 @@
+@import 'variables';
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  font-size: 14px;
+  border: none;
+  color: $color-white;
+  background-color: $color-button;
+  font-family: $font-roboto;
+  font-weight: $font-bold;
+  line-height: 16px;
+  margin: 0 16px 16px;
```

Comment:
Consider using padding instead of a fixed width and height to make the button more flexible and content-aware. Fixed dimensions can cause issues if the button's content changes or needs to be localized into different languages.